src/configure

```diff
@@ -514,8 +593,13 @@ function linux_configure_speex {
 }
 
 function fix_cxx_flag {
-  CXXCOMPILER=`grep "CXX = " kaldi.mk | awk '{print $3}'`
-  if [ $CXXCOMPILER=="g++" ]; then
+  USINGGXX=`grep -c "CXX = g++" kaldi.mk`
+  if [ $USINGGXX -ge 1 ]; then
+    if [ -z "$HOST" ] ; then
+      CXXCOMPILER="g++"
+    else
+      printf -v CXXCOMPILER "%s-g++" $HOST
```

Comment:
I think doing
CXXCOMPILER="$HOST-g++"
would be more readable, I had to look this up.